sql/conseil.sql

```diff
@@ -968,13 +968,25 @@ CREATE TABLE ethereum.tokens (
 );
 
 CREATE TABLE ethereum.token_transfers (
+  token_address text NOT NULL,
   block_number integer NOT NULL,
   transaction_hash text NOT NULL,
   from_address text NOT NULL,
   to_address text NOT NULL,
   value numeric NOT NULL
 );
 
+CREATE TABLE ethereum.tokens_history (
+    account_address text NOT NULL,
+    block_number integer NOT NULL,
+    transaction_hash text NOT NULL,
+    token_address text NOT NULL,
+    value numeric NOT NULL,
+    asof timestamp without time zone NOT NULL
+);
+
+CREATE INDEX ix_account_address ON ethereum.tokens_history USING btree (account_address);
```

Comment:
I would consider adding indexes also on `token_address` and maybe (`account_address`, `block_number`)? I think it could be useful - WDYT?

Reply:
It definitely makes sense to do it for `token_address` and `account_address`, though I'm not sure about `block_number`